hybridse/src/udf/udf.cc

```diff
@@ -157,6 +157,16 @@ int32_t weekofyear(Date *date) {
     }
 }
 
+void int_to_char(int32_t val, StringRef* output) {
+    if (val > 256)
+        val = val % 256;
+    output->size_ = 1;
+    char v = static_cast<char>(val);
+    std::string s;
+    s = c;
+    output->data_ = s.c_str();
```

Comment:
not correct, you should write data directly to the `AllocManagedStringBuf` returned buf.

Reply:
Are you sure? Maybe you can talk with @aceforeverd , he says that I'm not allowed to use another string!!!
![image](https://user-images.githubusercontent.com/27490425/170739914-bf381488-5afd-4444-bf90-bc3f43dd7c30.png)

Reply:
oh sorry, that's yourself. then what do you mean? could you give me an example? I'm not the staff of openMLDB, and I don't think I can understand what you say as accurately as your staff, especially you assign this issue as beginner's problem AFTER I take this PR, with so many problems (such as just one blank space, which needs nearly 2 hours to run this entire CI to ensure its correctness). @aceforeverd

Reply:
I mean you can write directly into the char* buf return from AllocManagedStringBuf. The string here will be freed after function exit, this make the returned StringRef invalid.

Again, pls be nice and patient

Reply:
ok